.eslintrc.js

```diff
@@ -1,26 +1,50 @@
 module.exports = {
   root: true,
-  extends: ['@metamask/eslint-config', '@metamask/eslint-config/config/jest', '@metamask/eslint-config/config/nodejs'],
-  ignorePatterns: ['!.eslintrc.js', '!jest.config.js', 'node_modules', 'dist', 'docs', 'coverage', '*.d.ts'],
+  extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
+  ignorePatterns: [
+    '!.eslintrc.js',
+    '!jest.config.js',
+    'node_modules',
+    'dist',
+    'docs',
+    'coverage',
+    '*.d.ts',
+  ],
   overrides: [
+    {
+      files: ['*.test.ts', '*.test.js'],
+      extends: ['@metamask/eslint-config-jest'],
+      rules: {
+        'jest/no-conditional-expect': 'off',
```

Comment:
Maybe worth copying the `// TODO: re-enble most of these rules` comment here too.